engine/runtime/src/main/scala/org/enso/interpreter/runtime/DefaultPackageRepository.scala

```diff
@@ -578,12 +578,10 @@ private class DefaultPackageRepository(
     val cache = ensurePackageIsLoaded(libraryName).toOption.flatMap { _ =>
       if (!loadedLibraryBindings.contains(libraryName)) {
         loadedPackages.get(libraryName).flatten.foreach(loadDependencies(_))
-        val cachedBindingOption = None
-        /* TBD: this has to be called somehow
-        context
-          .asInstanceOf[TruffleCompilerContext]
-          .deserializeLibraryBindings(libraryName)
-         */
+        val cachedBindingOption =
```

Comment:
Fixes the [deserialization problem](https://github.com/enso-org/enso/pull/8894/files#r1579122976) discovered by @hubertp.